core/frontend/helpers/ghost_head.js

```diff
@@ -44,10 +44,17 @@ function finaliseStructuredData(metaData) {
 }
 
 function getMembersHelper() {
-    return `
-        <script src="https://js.stripe.com/v3/"></script>
-        <script defer src="${getAssetUrl('public/members.js')}"></script>
-    `;
+    const stripePaymentProcessor = settingsCache.get('members_subscription_settings').paymentProcessors.find(
+        paymentProcessor => paymentProcessor.adapter === 'stripe'
+    );
+    const stripeSecretToken = stripePaymentProcessor.config.secret_token;
+    const stripePublicToken = stripePaymentProcessor.config.public_token;
+
+    var membersHelper = `<script defer src="${getAssetUrl('public/members.js')}"></script>`;
+    if (!!stripeSecretToken && stripeSecretToken !== '' && !!stripePublicToken && stripePublicToken !== '') {
```

Comment:
Think it would be cleaner to reuse `isPaymentConfigured` method logic straight from membersService https://github.com/TryGhost/Ghost/blob/3.2.0/core/server/services/members/index.js#L26-L29 :
```js
const settings = settingsCache.get('members_subscription_settings');
return !!settings && settings.isPaid && settings.paymentProcessors.length !== 0;
```

and leave a comment about where it's taken from. Also, no need for stripeSecretToken/stripePublicToken checks as we don't allow any other processors so far. YAGNI :smile:  

Ideally, we'd have a separate flag in the `settingsCache` or public settings endpoint which would identify if Stripe payment processor is configured, but that goes out of the scope of this problem :slightly_smiling_face: Reason being: using the settings directly couples the "frontend" with Ghost instance internals which we are trying to avoid (frontend is meant to be as independent as possible so that it can be run separately from Ghost API instance in the future). **But**, for now, think this is an acceptable approach if we leave a note for a person that comes around to refactor this code. 

//cc @allouis you might be interested :wink:

Reply:
@allouis just had another look at this logic in attempt to update the PR and realized the method I referenced in membersService above - `isPaymentConfigured` is never used, additionally we probably have a leftover flag `isPaid` hanging in the settings object as it's never changed to anything but it's default `false` value. Think it's safe to remove both `isPaid` flag and the `isPaymentConfigured` method?

Reply:
Additionally, we start an instance with `paymentProcessors` by [default](https://github.com/TryGhost/Ghost/blob/fa3686b/core/server/data/schema/default-settings.json#L210) just with empty values. So, I think, the solution here is the best one could have to check if payment is actually configured. Will merge this as is after little bit more testing and will clean up the flag and method mentioned above.